src/export/InstanceExporter.ts

```diff
@@ -27,11 +33,63 @@ export class InstanceExporter {
       );
 
       setPropertyOnInstance(instanceDef, pathParts, fixedValue);
+
+      // For each part of that path, we add fixed values from the SD
+      let path = '';
+      for (const [i, pathPart] of pathParts.entries()) {
+        path += `${path ? '.' : ''}${pathPart.base}`;
+        const element = instanceOfStructureDefinition.findElementByPath(path, this.resolve);
+        this.setFixedValuesForDirectChildren(element, pathParts.slice(0, i + 1), instanceDef);
+      }
     });
 
+    // Fix values from the SD for all elements at the top level of the SD
+    this.setFixedValuesForDirectChildren(
+      instanceOfStructureDefinition.findElement(instanceDef.resourceType),
+      [],
+      instanceDef
+    );
+
     return instanceDef;
   }
 
+  /**
+   * Given an ElementDefinition, set fixed values for the direct children of that element
+   * according to the ElementDefinitions of the children
+   * @param {ElementDefinition} element - The element whose children we will fix
+   * @param {PathPart[]} existingPath - The path to the element whose children we will fix
+   * @param {InstanceDefinition} instanceDef - The InstanceDefinition to fix values on
+   */
+  private setFixedValuesForDirectChildren(
+    element: ElementDefinition,
+    existingPath: PathPart[],
+    instanceDef: InstanceDefinition
+  ) {
+    const directChildren = element
+      .children()
+      .filter(c => c.path.split('.').length === element.path.split('.').length + 1);
```

Comment:
We may need something like this in the future.  I think it might make sense to move this into the `StructureDefinition` class.  One approach would be to add a new optional `excludeGrandchildren` (or `directOnly`) argument to the `children` function.

Reply:
This seemed fair so I added it